core/domain/prod_validation_jobs_one_off.py

```diff
@@ -30,15 +29,21 @@ class BaseModelValidator(object):
     """Base class for validating models."""
 
     errors = {}
+    # external_models is a map that's keyed by field name. Each value consists
+    # of the model class and a list of (external_key, external_model) tuples.
```

Comment:
external_model --> external_model_instance

Reply:
Oh, in my earlier comment, I meant in `(external_key, external_model)`. (But your renaming is good too.)